tests/configlet/util/configlet.py

```diff
@@ -18,10 +18,13 @@ def get_pfc_time():
     ret = 0
     pfc_wd = config_db_data_wo_t0.get("PFC_WD", {})
     for n, val in pfc_wd.items():
-        ret = int(val["detection_time"])
+        ret = int(val.get("detection_time", 0))
+        if ret:
```

Comment:
if not ret?

Reply:
This function will return 0; The PFC_WD entry for this port will not be added, if 0.

```
        if pfc_time:
            # "PFC_WD"
            pfc_wd[local_port] = {
                    "action": "drop",
                    "detection_time": pfc_time,
                    "restoration_time": pfc_time }
```